core/src/main/java/hudson/Functions.java

```diff
@@ -1326,7 +1329,9 @@ protected int compare(long idA, long idB) {
         }
     }
 
-    public static class ThreadGroupMap extends ThreadSorterBase implements Comparator<ThreadInfo> {
+    public static class ThreadGroupMap extends ThreadSorterBase implements Comparator<ThreadInfo>, Serializable {
+
+        private static final long serialVersionUID = 7803975728695308444L;
```

Comment:
For some reason, the standard in this code base is to default the `serialVersionUID` to `1L` instead of some randomly generated number. The reason for it is unknown to me, but I've gotten scolded for this by others in the past :)